pkg/ebpf/c/missing_definitions.h

```diff
@@ -4,88 +4,88 @@
 // NOTE: vmlinux.h contains kernel BTF information but not macros.
 
 #ifndef __TRACEE_MISSING_MACROS_H__
-#define __TRACEE_MISSING_MACROS_H__
```

Comment:
With latest changes, fix-fmt should be applied to all .c and .h files on the tree. This also fixes issues making the PR not to pass the check-fmt command.